packages/react-components/src/templates/__tests__/UserProfileResearch.test.tsx

```diff
@@ -115,7 +112,7 @@ it('renders an edit button for the role on the team (REGRESSION)', () => {
           id: '42',
           displayName: 'Team',
           role: 'Lead PI (Core Leadership)',
-          editHref: '/edit-team-membership/42',
+          editHref: '/edit-role/42',
```

Comment:
this is no longer associated to the team, there is only one edit-role link
we can remove this line from here